styles.css

```diff
@@ -0,0 +1,654 @@
+}
+
+#featured-speakers h4 {
+  text-align: center;
+  font-size: 22px;
+  font-family: sans-serif;
+  font-weight: 400;
+  margin: 25px;
+  text-shadow: 0.5px 0 black;
+}
+
+#featured-speakers .about-speakers {
+  display: grid;
+  grid-template: auto auto auto / auto auto;
+  gap: 10px;
+  column-gap: 25px;
+  font-weight: 600;
+  justify-content: center;
+  align-items: center;
+}
+
+.about-speakers .face {
+  grid-area: 1 / 1 / 4 / 2;
+}
+
+.about-speakers .pink {
+  color: rgb(230, 10, 10);
+}
+
+.partners {
+  display: none;
+}
+
+/* styling the about html file  */
+
+#home-about {
+  width: 100%;
+  text-align: center;
+  font-weight: 600;
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+
+  /* justify-content: center; */
+
+  /* align-items: center; */
+  margin: 0 auto;
+}
+
+#home-about #about-us {
+  background-color: whitesmoke;
+  background-image: url('https://catherineasquithgallery.com/uploads/posts/2021-03/1614791986_122-p-fon-dlya-delovoi-prezentatsii-129.jpg');
+}
+
+#home-about .hello {
+  color: rgb(238, 7, 7);
+  font-weight: 700;
+}
+
+#home-about h1 {
+  color: rgba(247, 12, 12, 0.991);
+  font-size: 40px;
+  font-weight: 700;
+  font-family: sans-serif;
+  margin-top: 100px;
+}
+
+#home-about .about-course {
+  margin: 35px;
+  padding: 25px 15px;
+  border-radius: 5px;
+  line-height: 28px;
+  background-color: white;
+}
+
+#home-about .about-course-dskt {
+  display: none;
+}
+
+#home-about .contact {
+  margin: 10px 80px;
+  line-height: 25px;
+}
+
+.contact strong {
+  font-family: sans-serif;
+}
+
+/* #about-logo {
+  background-color: white;
+} */
+
+#about-logo h3 {
+  font-family: sans-serif;
+  font-weight: 400;
+  font-size: 30px;
+  margin: 30px 100px;
+  padding: 10px;
+}
+
+#about-logo .logo-mob {
+  margin: 0 70px;
+  line-height: 26px;
+}
+
+#about-logo .logo-dskt {
+  display: none;
+}
+
+#about-time {
+  padding: 40px;
+}
+
+#about-time img {
+  border: 1px solid grey;
+  width: 100%;
+  padding: 30px 50px;
+}
+
+/* #about-time h2 {
+  font-family: sans-serif;
+  font-size: 35px;
+}
+
+#about-time p {
+  font-size: 13px;
+  border-top: 1px solid grey;
+  border-bottom: 1px solid grey;
+} */
+
+#past-courses {
+  border-top: 1px solid grey;
+}
+
+#past-courses h3 {
+  font-family: sans-serif;
+  font-size: 22px;
+  margin: 25px 50px;
+}
+
+#past-courses p {
+  margin: 25px 40px;
+}
+
+#past-courses img {
+  width: 80%;
+  margin-bottom: 20px;
+}
+
+#home-about .partners {
+  display: grid;
+  grid-template: 70px 10px 100px 100px / repeat(3, auto);
+  text-align: center;
+  justify-content: space-around;
+  align-items: center;
+  background-color: #414246;
+  margin: 20px 0;
+}
+
+.partners h2 {
+  color: white;
+  font-size: 28px;
+}
+
+.partners .partner-logo {
+  width: 100px;
+}
+
+.partners .item0 {
+  grid-area: 1 / 1 / 2 / 4;
+}
+
+.partners .item1 {
+  grid-area: 2 / 1 / 3 / 4;
+  margin: 0 auto;
+}
+
+.partners .item6 {
+  grid-area: 4 / 3 / 5 / 4;
+}
+
+#footer {
```

Comment:
Please add space to the left of the footer section like the marked on below.
<img width="557" alt="Screenshot 2023-02-09 at 9 42 39 PM" src="https://user-images.githubusercontent.com/25887253/217907724-67627e46-2ccf-4a8a-a006-e5a5db0a1b43.png">

<img width="557" alt="Screenshot 2023-02-09 at 9 43 28 PM" src="https://user-images.githubusercontent.com/25887253/217907705-27b5512d-b4a5-41dc-9fe2-0eaba1899d56.png">